mlir/include/mlir/Dialect/Tosa/Transforms/TosaFoldCommon.h

```diff
@@ -0,0 +1,60 @@
+//===- TosaFoldCommon.h - Helper Functions for Folds ------------*- C++ -*-===//
```

Comment:
Future PRs will bring `applyElementWise` for binary and ternary (?) ops, and I had re-used them in another project because they essentially allow to perform computations on two Attributes taking into account tosa-style broadcasting, which is pretty useful.

The other functions might move to mlir/lib/Dialect/Tosa/Transforms